src/js/http.js

```diff
@@ -13,17 +13,43 @@
  * limitations under the License.
  */
 
-var Server = require('http_server').Server;
+var net = require('net');
 var ClientRequest = require('http_client').ClientRequest;
 var HTTPParser = require('httpparser');
+var HTTPServer = require('http_server');
```

Comment:
"http underscore server" and "httpparser" sounds inconsistent. Although this is legacy so you can keep it.

Reply:
Yes, It bothers me too. :) I didn't want to change it in this PR, because it is quite unrelated.